### DISCUSSION TITLE
https://cucoders.dev/publicaciones/andyromerodev/el-estado-del-c-digo-abierto-en-github/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#47] [Page Comments]
# https://cucoders.dev/publicaciones/andyromerodev/el-estado-del-c-digo-abierto-en-github/

En un artículo anterior explicaba el impacto social del «Código abierto» y su importancia para la sociedad. Hoy les traigo El estado del código abierto en GitHub según el sitio oficial Octoverse de Github.

https://cucoders.dev/publicaciones/andyromerodev/el-estado-del-c-digo-abierto-en-github/

**Comment by manuelernestog** [comment]
Interesantísimo como VScode tiene tantos contribuidores 🤯
